03_launch_mgmt_cluster.sh

```diff
@@ -172,8 +134,6 @@ function update_images(){
     IMAGE_NAME="${IMAGE##*/}"
     LOCAL_IMAGE="${REGISTRY}/localimages/$IMAGE_NAME"
     OLD_IMAGE_VAR="${IMAGE_VAR%_LOCAL_IMAGE}_IMAGE"
-    # Strip the tag for image replacement
-    OLD_IMAGE="${!OLD_IMAGE_VAR%:*}"
```

Comment:
Not clear why are we removing this?

Reply:
This was used to avoid touching the tag of the image. Now I want to replace also the tag so there is no point in stripping it.
So previously it did something like this:
```
my-registry.org/some/image:tag -> local.registry/localimages/some/image:tag
```
Now it will replace all of it (based on variables like `BARE_METAL_OPERATOR_IMAGE`):
```
my-registry.org/some/image:tag -> local.registry/localimages/some/custom-image:custom-tag
```

Reply:
ok I got it. It looks fine then